cli/pazuzu/commands.go

```diff
@@ -3,31 +3,91 @@ package main
 import (
 	"bufio"
 	"fmt"
-	"log"
-	"os"
-
 	"github.com/urfave/cli"
 	"github.com/zalando-incubator/pazuzu"
+	"log"
+	"os"
 	"regexp"
 	"text/tabwriter"
 )
 
 var cnfGetCmd = cli.Command{
 	Name:  "get",
 	Usage: "Get pazuzu configuration",
-	Action: func(с *cli.Context) error {
-		// log.Print("Getting pazuzu configuration")
-		// return nil
-		return ErrNotImplemented
+	Action: func(c *cli.Context) error {
+		a := c.Args()
+		if len(a) != 1 {
+			return ErrTooFewOrManyParameters
+		}
+		//
+		givenPath := a.Get(0)
+		cfgMirror := pazuzu.GetConfigMirror()
+		repr, err := cfgMirror.GetRepr(givenPath)
+		if err == nil {
+			fmt.Println(repr)
+			return nil
+		}
+		return ErrNotFound
 	},
 }
+
 var cnfSetCmd = cli.Command{
 	Name:  "set",
 	Usage: "Set pazuzu configuration",
-	Action: func(с *cli.Context) error {
-		// log.Print("Setting pazuzu configuration")
-		// return nil
-		return ErrNotImplemented
+	Action: func(c *cli.Context) error {
+		a := c.Args()
+		if len(a) != 2 {
+			return ErrTooFewOrManyParameters
+		}
+		//
+		givenPath := a.Get(0)
+		givenValRepr := a.Get(1)
+		cfg := pazuzu.GetConfig()
+		cfgMirror := pazuzu.GetConfigMirror()
+		errSet := cfgMirror.SetConfig(givenPath, givenValRepr)
+		if errSet == nil {
+			// Oh, it's nice.
+			_ = cfg.Save()
+			return nil
+		}
+		fmt.Printf("FAIL [%v]\n", errSet)
+		return ErrNotFound
+	},
+}
+
+var cnfHelpCmd = cli.Command{
+	Name:  "help",
+	Usage: "Print help on configuration",
+	Action: func(c *cli.Context) error {
+		cfgMirror := pazuzu.GetConfigMirror()
+		fmt.Println("Pazuzu CLI Config related commands:")
+		fmt.Println("\tpazuzu config list\t -- Listing of configuration.")
+		fmt.Println("\tpazuzu config help\t-- This help documentation.")
+		fmt.Println("\tpazuzu config get KEY\t-- Get specific configuration value.")
+		fmt.Println("\tpazuzu config set KEY VALUE\t-- Set configuration.")
+		fmt.Printf("\nConfiguration keys and its descriptions:\n")
```

Comment:
Most probably this can be implemented using subcommands functionality. 

You can find few details here: https://github.com/urfave/cli#subcommands